src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersMappingBuilderContext.cs

```diff
@@ -66,6 +75,40 @@ private HashSet<string> InitIgnoredUnmatchedProperties(IEnumerable<string> allPr
         return unmatched;
     }
 
+    private IEnumerable<string> GetIgnoredObsoleteTargetMembers()
+    {
+        var obsoleteStrategy = BuilderContext.Configuration.Properties.IgnoreObsoleteMembersStrategy;
+
+        if (obsoleteStrategy.HasFlag(IgnoreObsoleteMembersStrategy.Target))
+        {
+            var obsoleteAttribute = BuilderContext.Types.Get<ObsoleteAttribute>();
+
+            return Mapping.TargetType
+                .GetAccessibleMappableMembers()
+                .Where(x => x.MemberSymbol.HasAttribute(obsoleteAttribute))
+                .Select(x => x.Name);
+        }
+
+        return Enumerable.Empty<string>();
```

Comment:
I'd invert the if condition, same for `GetIgnoredObsoleteSourceMembers` (IMO improves the readability).